src/test/txreconciliation_tests.cpp

```diff
@@ -81,4 +81,75 @@ BOOST_AUTO_TEST_CASE(IsPeerRegisteredTest)
     BOOST_CHECK(!tracker.IsPeerRegistered(peer_id0));
 }
 
+BOOST_AUTO_TEST_CASE(ShouldFanoutToTest)
```

Comment:
I think a test can be added to exercise `AddToSet` and the check of the boundary `MAX_SET_SIZE` value.